src/base/tests/test_util.py

```diff
@@ -1268,31 +1268,31 @@ def test_ensure_xmlid_match_record(self):
 
     @unittest.skipUnless(util.version_gte("16.0"), "Only work on Odoo >= 16")
     def test_replace_in_all_jsonb_values(self):
-        test_partner_title = self.env["res.partner.title"].create(
+        test_partner_category = self.env["res.partner.category"].create(
```

Comment:
The column must be jsonb, aka a translated field. I'd propose `res.country.group.name`